### DISCUSSION TITLE
【求助】怎么可以不关注某个帖子啊55，一评论邮箱就被轰炸，求大佬指点

### DISCUSSION THREAD

**Discussion by shengwangyonghu** [share121/inter-knot#2418] [其他]
![93da1219c6ab7da8d563c9a7f33257e](https://github.com/user-attachments/assets/5343e4cb-fbe6-4f5b-a3cd-93ea12e4af79)
还有我在手机上下载了github为啥登陆的网页是黑屏啊

**Comment by shengwangyonghu** [comment]
还有咋改绳网名字啊，这个感觉好呆

**Reply by 6078HITHEDAY** [reply]
改GitHub的用户名

**Comment by shengwangyonghu** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >求

**Comment by Dave-12138** [comment]
![点unsubscribe](https://github.com/user-attachments/assets/939cdd50-65a5-4efc-810e-f812ff7682c7)

点unsubscribe.

**Reply by shengwangyonghu** [reply]
感谢感谢
